lib/atoms/title1/example.html

```diff
@@ -0,0 +1 @@
+<ln-atom-title1 title="{{title}}" class="{{class}}"></ln-atom-title1>
```

Comment:
This is a very basic example, but maybe the title component will include other tags in the future (underline, subtitile, icon, etc). Maybe in the future this title component is not just an h1 tag.